release_notes/ocp-4-15-release-notes.adoc

```diff
@@ -2797,6 +2797,34 @@ This section will continue to be updated over time to provide notes on enhanceme
 For any {product-title} release, always review the instructions on xref:../updating/updating_a_cluster/updating-cluster-web-console.adoc#updating-cluster-web-console[updating your cluster] properly.
 ====
 
+// 4.15.55
+[id="ocp-4-15-55_{context}"]
+=== RHSA-2025:11351 - {product-title} 4.15.55 bug fix update
+
+Issued: 23 July 2025
+
+{product-title} release 4.15.55 is now available. The list of bug fixes that are included in this update is documented in the link:https://access.redhat.com/errata/RHSA-2025:11351[RHSA-2025:11351] advisory. The RPM packages that are included in this update are provided by the link:https://access.redhat.com/errata/RHSA-2025:11352[RHSA-2025:11352] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory.
+
+You can view the container images in this release by running the following command:
+
+[source,terminal]
+----
+$ oc adm release info 4.15.55 --pullspecs
+----
+
+[id="ocp-4-15-55-bug-fixes_{context}"]
+==== Bug fixes
+
+* Before this update, the `must-gather` tool collected CPU-related performance data sequentially per node, leading to significantly longer collection times in larger clusters. With this release, the `must-gather` tool collects the data in parallel, resulting in a reduction in collection time. (link:https://issues.redhat.com/browse/OCPBUGS-36371[OCPBUGS-36371])
+
+* Before this update, cluster installations in affected versions hung due to missing API endpoint certificates. As a result, installations failed or stalled, delaying cluster setup and service deployment. With this update, the certificate issue in cluster installations in {hcp} is resolved. This fix prevents API endpoint errors and allows {rh-rhacm-first} agents to start successfully. Cluster installations now complete as expected, with a smoother setup experience. (link:https://issues.redhat.com/browse/OCPBUGS-58506[OCPBUGS-58506])
```

Comment:
```suggestion
* Before this update, in {hcp}, hosted cluster installations in affected versions hung due to missing API endpoint certificates. As a result, cluster installations failed or stalled, delaying cluster setup and service deployment. With this update, the certificate issue in cluster installations in {hcp} is resolved. This fix prevents API endpoint errors and allows {rh-rhacm-first} agents to start successfully. Cluster installations now complete as expected, with a smoother setup experience. (link:https://issues.redhat.com/browse/OCPBUGS-58506[OCPBUGS-58506])
```

Just one minor comment to clarify in the beginning of the bug text that it's related to clusters in HCP.